web/styles/dartdoc/_signature.scss

```diff
@@ -90,20 +86,21 @@
 
 /* Note that the generated HTML for pub packages may have `section` tags
    transformed into `div` tags, so we have two selectors here. */
-section.multi-line-signature div.parameters,
-div.multi-line-signature div.parameters {
-  margin-left: 24px;
-}
-
-#instance-methods dt.inherited .name,
-#instance-properties dt.inherited .name,
-#operators dt.inherited .name {
-  font-weight: 400;
-  font-style: italic;
-}
-
-#instance-methods dt.inherited .signature,
-#instance-properties dt.inherited .signature,
-#operators dt.inherited .signature {
-  font-weight: 400;
+section, div {
+  .multi-line-signature div.parameters {
+    margin-left: 24px;
+  }
+}
+
+#instance-methods, #instance-properties, #operators {
+  dt.inherited {
+    .name {
+      font-weight: 400;
+      font-style: italic;
+    }
+
+    .signature {
+      font-weight: 400;
+    }
+  }
```

Comment:
It's funny how this is 1 line longer, even with the de-duplication :P but it's still simpler and more maintainable.